doc/source/serve/direct-ingress.md

```diff
@@ -0,0 +1,101 @@
+# Direct Ingress
+
+In the 2.1, Serve provides alpha version gRPC ingress. In this section, you will learn how to 
+
+* use Serve internal gRPC schema to send traffic
+* bring your own gRPC schema into serve application
+
+## Use Serve Schema
+
+Internally, serve provides a simple gRPC schema.
+```
+message PredictRequest {
+  map<string, bytes> input = 2;
+}
+
+message PredictResponse {
+  bytes prediction = 1;
+}
+
+service PredictAPIsService {
+  rpc Predict(PredictRequest) returns (PredictResponse);
+}
+```
+
+Code example:
+
+Server
+```{literalinclude} ../serve/doc_code/direct_ingress.py
+:start-after: __begin_server__
+:end-before: __end_server__
+:language: python
+```
+
+Client
+```{literalinclude} ../serve/doc_code/direct_ingress.py
+:start-after: __begin_client__
+:end-before: __end_client__
+:language: python
+```
+
+:::{note}
+* `input` is a dictionary of `map<string, bytes> ` following the schema described
```

Comment:
```suggestion
* `input` is a dictionary of `map<string, bytes> ` following the schema described above.
```

Reply:
In our discussion offline I remember we mentioned something like this is a standard interface where the strings are used for features.  Is it worth mentioning anything about that in the docs?

Reply:
Cool, makes sense